src/components/Popup.js

```diff
@@ -17,15 +17,13 @@ export default class Popup {
 
   _handleEscClose = (e) => {
     if (e.key === "Escape") {
-      const modal = document.querySelector(".modal_opened");
-      this.close(modal);
+      this.close();
     }
   };
 
   _handleMouseClickModalClose(e) {
```

Comment:
Please, delete `_handleMouseClickModalClose` because you set the overlay handler in line 35